base/initdefs.jl

```diff
@@ -136,10 +136,10 @@ function init_load_path()
     end
     project = (JLOptions().project != C_NULL ?
         unsafe_string(Base.JLOptions().project) :
```

Comment:
This should probably be handled in init.c, for conisistency with all other options that define paths

Reply:
The `abspath` call? Like https://github.com/JuliaLang/julia/blob/0846eb56548a52f07998f5ee354562fbff85a8f7/src/init.c#L587-L594?

Reply:
We still need the `abspath` here for relative paths from `JULIA_PROJECT` though.